pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -648,11 +679,16 @@ func (exp *FlowExporter) fillEgressInfo(conn *flowexporter.Connection) {
 	conn.EgressName = egressName
 	conn.EgressIP = egressIP
 	conn.EgressNodeName = egressNodeName
-	klog.V(4).InfoS("Filling Egress Info for flow", "Egress", conn.EgressName, "EgressIP", conn.EgressIP, "EgressNode", conn.EgressNodeName, "SourcePod", klog.KRef(conn.SourcePodNamespace, conn.SourcePodName))
+	if klog.V(5).Enabled() {
+		klog.InfoS("Filling Egress Info for flow", "Egress", conn.EgressName, "EgressIP", conn.EgressIP, "EgressNode", conn.EgressNodeName, "SourcePod", klog.KRef(conn.SourcePodNamespace, conn.SourcePodName))
+	}
 }
 
 func (exp *FlowExporter) exportConn(conn *flowexporter.Connection) error {
 	conn.FlowType = exp.findFlowType(*conn)
+	if conn.FlowType == flowTypeUnsupported {
```

Comment:
So we will export the flowTypeUnknown flow anyway? or it's actually not possible when the FlowExporter is running? It seems the nodeRouteController will be nil only when it's running as a VM agent, Do we support FlowExporter when Antrea is running as a VM agent?

Reply:
I preserved the existing behavior, we can revisit it later.